abs.yaml

```
@@ -0,0 +1,6 @@
+# dependence on linux sysroot, eg lib64/libc.so.6, don't do overlinking checks
+build_parameters:
+  - "--suppress-variables"
```

```suggestion
  - "--suppress-variables"
  - "--skip-existing"
```
To be on the safe side.